charts/rancher-istio/1.4.400/requirements.yaml

```diff
@@ -1,40 +1,40 @@
 dependencies:
   - name: sidecarInjectorWebhook
-    version: 1.4.3
+    version: 1.4.4
     condition: sidecarInjectorWebhook.enabled
   - name: security
-    version: 1.4.3
+    version: 1.4.4
     condition: security.enabled
   - name: gateways
-    version: 1.4.3
+    version: 1.4.4
     condition: gateways.enabled
   - name: mixer
-    version: 1.4.3
+    version: 1.4.4
     condition: or mixer.policy.enabled mixer.telemetry.enabled
   - name: nodeagent
-    version: 1.4.3
+    version: 1.4.4
     condition: nodeagent.enabled
   - name: pilot
-    version: 1.4.3
+    version: 1.4.4
     condition: pilot.enabled
   - name: grafana
-    version: 1.4.3
+    version: 1.4.4
     condition: grafana.enabled
   - name: prometheus
-    version: 1.4.3
+    version: 1.4.4
     condition: prometheus.enabled
   - name: tracing
-    version: 1.4.3
+    version: 1.4.4
     condition: tracing.enabled
   - name: galley
-    version: 1.4.3
+    version: 1.4.4
     condition: galley.enabled
   - name: kiali
-    version: 1.4.3
+    version: 1.9.0
```

Comment:
Kiali chart doesn't use current istio version

Reply:
hmm, this version doesn't mean image version, correct? It means chart version.

Reply:
Correct, although the image doesn't match either. Its pointing at sub-chart version, which wasn't matching on 1.4.300 either: https://github.com/rancher/system-charts/blob/dev/charts/rancher-istio/1.4.300/charts/kiali/Chart.yaml#L7

Reply:
reverted to keep in line with upstream